From 6c4ebfbbb4fec95f311570fb487885e7d1e26dd2 Mon Sep 17 00:00:00 2001
From: Arnd Bergmann <arnd@arndb.de>
Date: Wed, 6 Dec 2017 09:50:20 -0800
Subject: [PATCH 010/205] memory: ti-emif-sram: remove unused variable

This commit comes from:
  git://git.ti.com/processor-sdk/processor-sdk-linux.git

The newly introduced driver causes a harmless warning for a variable
that was evidently never used:

drivers/memory/ti-emif-pm.c: In function 'ti_emif_remove':
drivers/memory/ti-emif-pm.c:303:17: error: unused variable 'dev' [-Werror=unused-variable]

Fixes: 8428e5ad750d ("memory: ti-emif-sram: introduce relocatable suspend/resume handlers")
Signed-off-by: Arnd Bergmann <arnd@arndb.de>
Signed-off-by: Santosh Shilimkar <santosh.shilimkar@oracle.com>
(cherry picked from commit 84a1695a6f925291f301951498fd7b56d3cb40a2)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/memory/ti-emif-pm.c |    1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/memory/ti-emif-pm.c b/drivers/memory/ti-emif-pm.c
index 4ea1514..62a86c4 100644
--- a/drivers/memory/ti-emif-pm.c
+++ b/drivers/memory/ti-emif-pm.c
@@ -300,7 +300,6 @@ static int ti_emif_probe(struct platform_device *pdev)
 
 static int ti_emif_remove(struct platform_device *pdev)
 {
-	struct device *dev = &pdev->dev;
 	struct ti_emif_data *emif_data = emif_instance;
 
 	emif_instance = NULL;
-- 
1.7.9.5

